pkgs/data/themes/andromeda-gtk-theme/default.nix

```
@@ -0,0 +1,27 @@
+{ lib, stdenvNoCC, fetchFromGitHub }:
+stdenvNoCC.mkDerivation {
+  pname = "andromeda-gtk-theme";
+  version = "250751a";
```

```suggestion
  version = "0-unstable-2024-03-04";
```
See https://github.com/NixOS/nixpkgs/tree/master/pkgs#conventions